adaptive/learner/learner1D.py

```diff
@@ -580,22 +580,33 @@ def _loss(self, mapping, ival):
         loss = mapping[ival]
         return finite_loss(ival, loss, self._scale[0])
 
-    def plot(self):
+    def plot(self, *, scatter_or_line=None):
         """Returns a plot of the evaluated data.
 
         Returns
         -------
         plot : `holoviews.element.Scatter` (if vdim=1)\
                else `holoviews.element.Path`
             Plot of the evaluated data.
+        scatter_or_line : str, optional
```

Comment:
This is a parameter, not a return value

Reply:
also set the default value to `scatter` and mention this in the docstring

Reply:
Done!